### PR TITLE
ZEN-28507 Global config option for command/python/etc datasources mis…

### PR DESCRIPTION
**Pull request by romanbakaleyko**
…sing

### CODE REVIEW

**Comment by romanbakaleyko** [comment]
demo in the ticket ZEN-28507

File: Products/ZenModel/RRDDataSource.py [hunk 1]
```diff
@@ -83,6 +83,22 @@ class RRDDataSource(ZenModelRM, ZenPackable):
     security = ClassSecurityInfo()
 
     
+    def talesEval(self, text, context):
```
Comment (jplouis) [inline Products/ZenModel/RRDDataSource.py]: Where is this being used?

File: Products/ZenHub/services/CommandPerformanceConfig.py [hunk 2]
```diff
@@ -73,7 +73,7 @@ def _getDsDatapoints(self, device, comp, ds, ploader, perfServer):
     def _getDsCycleTime(self, comp, templ, ds):
         cycleTime = 300
         try:
-            cycleTime = int(ds.cycletime)
+            cycleTime = int(ds.getCycleTime(comp))
```
Comment (jplouis) [inline Products/ZenHub/services/CommandPerformanceConfig.py]: Where does getCycleTime come from?

Comment (romanbakaleyko) [reply Products/ZenHub/services/CommandPerformanceConfig.py]: updated

File: Products/ZenModel/RRDDataSource.py (hunk 1 above)
Comment (romanbakaleyko) [reply Products/ZenModel/RRDDataSource.py]: updated

**Comment by romanbakaleyko** [comment]
Closing while Thebe release in in progress.

**Pull request closed without merging**